packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -82,8 +90,8 @@ class DependencyExtractionWebpackPlugin {
 				: defaultRequestToExternal( request );
 		}
 
-		if ( this.useModules && externalRequest === true ) {
-			externalRequest = request;
+		if ( externalRequest instanceof Error ) {
+			return callback( externalRequest );
```

Comment:
It's best not to throw the error directly. If we throw it directly webpack doesn't know where the error is coming from and shows a general error.

`throw`:
```
ERROR in Attempted to use WordPress script in a module: @wordpress/api-fetch
```

`callback( err )`:
```
ERROR in ./src/index.js 1:0-23
Module not found: Error: Attempted to use WordPress script in a module: lodash
```

Another option would be to throw the error in `defaultRequestToExternal` and use `try/catch` to pass the error to the callback.